source/deployment-options/deploying-with-ansible/guide/install-indexer-dashboard.rst

```diff
@@ -210,18 +210,30 @@ Now, It seems that we are ready to run the playbook and start the installation.
 
 #. We can check the status of our new services on our Wazuh indexer and dashboard server.
 
-   -  Wazuh indexer.
+   -  **Wazuh indexer**
 
       .. code-block:: console
 
          # systemctl status wazuh-indexer
 
-   -  Wazuh dashboard
+   -  **Wazuh dashboard**
     
       .. code-block:: console
 
          # systemctl status wazuh-dashboard
 
+
+.. note::
+	
+	- 	The Wazuh dashboard can be accessed by visiting ``https://<wazuh_server_IP>:443``
+
+	- 	The default credentials for Wazuh deployed using ansible is:
+		
+		|	Username: admin
```

Comment:
Is it indented for any reason?

Reply:
I based this addition on the comment of @alberpilot: https://github.com/wazuh/wazuh-documentation/issues/5854#issuecomment-1367887846.